vault/resource_identity_oidc_assignment.go

```diff
@@ -0,0 +1,122 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+const identityOIDCAssignmentPathTemplate = "identity/oidc/assignment"
```

Comment:
nit: this feels more like a pure path since it doesn't contain any interpolation variables, may want to consider renaming it.

Reply:
Updated to `Prefix` instead of `Template`